app/components/Actions.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 import styled from "styled-components";
 import breakpoint from "styled-components-breakpoint";
-import Flex from "shared/components/Flex";
+import Flex from "components/Flex";
```

Comment:
There are no more shared React components, so moved everything across to the main `components` directory.